docs/iceberg-rest-service.md

```diff
@@ -12,62 +12,75 @@ The Apache Gravitino Iceberg REST Server follows the [Apache Iceberg REST API sp
 - When writing to HDFS, the Gravitino Iceberg REST catalog service can only operate as the specified HDFS user and
   doesn't support proxying to other HDFS users. See [How to access Apache Hadoop](gravitino-server-config.md#how-to-access-apache-hadoop) for more details.
 
 :::info
-Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `1` by default.
 Builds with Hadoop 2.10.x. There may be compatibility issues when accessing Hadoop 3.x clusters.
 :::
 
-## Apache Gravitino Iceberg REST catalog service configuration
+## Server management
 
-Assuming the Gravitino server is deployed in the `GRAVITINO_HOME` directory, you can locate the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md). There are four configuration properties for the Iceberg REST catalog service:
+Iceberg REST server could start as a standalone server or an auxiliary service in Gravitino server. There are some differences like below:
 
-1. [**REST Catalog Server Configuration**](#rest-catalog-server-configuration): you can specify the HTTP server properties like host and port.
+|                         | Standalone server                                                                                                                              | Auxiliary service in Gravitino server                                                                                             | 
+|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| How to build package    | `./gradlew compileIcebergRESTServer -x test` build a standalone Iceberg REST server in `distribution/gravitino-iceberg-rest-server` directory. | `./gradlew compileDistribution -x test` build Gravitino server and Iceberg REST server in `distribution/package` directory.       | 
+| How to assemble package | `./gradlew assembleIcebergRESTServer -x test` build a tar file with Iceberg REST server only in `distribution` directory.                      | `./gradlew assembleDistribution -x test` build a tar file with Gravitino server and Iceberg REST server `distribution` directory. | 
+| How to start server     | `./bin/gravitino-iceberg-rest-server.sh start`                                                                                                 | `./bin/gravitino.sh start`                                                                                                        | 
+| Configuration files     | `./conf/gravitino-iceberg-rest-server.conf`, `conf/log4j2.properties`                                                                          | `./conf/gravitino.conf`, `conf/log4j2.properties`                                                                                 | 
```

Comment:
I think we should also add these "how to build", "how to start" to the relative docs.

Reply:
Also here.